packages/repo-metadata-lint/src/validate.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import {URL} from 'url';
+
+export interface ValidationResult {
+  status: 'success' | 'error';
+  errors: Array<string>;
+}
+
+const LIBRARY_TYPES = [
+  'GAPIC_AUTO',
+  'GAPIC_MANUAL',
+  'AGENT',
+  'MISC',
+  'AUTH',
+  'various',
```

Comment:
'various' is not a member of the library type set (apols, my doc is  ambiguous here)

Reply:
Got rid of `various` 👍